OrbitQt/orbitmainwindow.cpp

```diff
@@ -72,8 +73,24 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
 
   GOrbitApp->AddCaptureStartedCallback(
       [this] { ui->HomeTab->setDisabled(true); });
-  GOrbitApp->AddCaptureStoppedCallback(
-      [this] { ui->HomeTab->setDisabled(false); });
+
+  auto finalizing_capture_dialog = std::make_shared<QProgressDialog>(
+      "Waiting for the remaining capture data...", "OK", 0, 0, this, Qt::Tool);
```

Comment:
One more comment: You are passing `this` as the 5th argument (parent). This means in Qt you're transfering ownership of this object to the parent. But at the same time you have a shared_ptr holding the ownership. This might lead to a double-free situation when the shared_ptr is destructed after `this` which is probably not the case here but should be avoided even though.

One solution is to just pass `nullptr` instead of `this` as a parent. But that might affect the position of the dialog. Another solution is to use a raw pointer of a QPointer and construct the dialog with a plain `new`. Not so nice, I know.

Reply:
You're absolutely right, thanks.

So I tried passing `nullptr` and the position is not too bad, as it's always the center of the screen, but that's conceptually wrong, so I guess I'll go with `new`, unfortunately it seems to be somewhat the norm with Qt.